benchmarking/tpch/ray_job_runner.py

```diff
@@ -1,3 +1,11 @@
+# /// script
+# requires-python = ">=3.12"
+# dependencies = [
+#   "ray[default]",
+#   "getdaft",
+# ]
+# ///
+
```

Comment:
This is just to make it possible to run this via `uv run benchmarking/tpch/ray_job_runner.py`.